lib/src/base-styles/_base.scss

```diff
@@ -41,4 +41,5 @@
 @use 'nx-system-notice';
 @use 'nx-divider';
 @use 'nx-text-link';
+@use 'nx-toast';
```

Comment:
`NxToast` is only usable as a React component, not as pure styles.  This means that the correct way to imports its styles is by importing its SCSS file from its TSX file, not here.

Reply:
> `NxToast` is only usable as a React component, not as pure styles. This means that the correct way to imports its styles is by importing its SCSS file from its TSX file, not here.

SCSS for Toast moved and imported via NxToastContainer.tsx here: [846e5d3](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/846e5d3d64a19ce1fd1d4f44d3a2726eccafe642)